src/app/product/components/product-detail-page/write-product-review/write-product-review.component.scss

```diff
@@ -0,0 +1,48 @@
+@import '../../../../shared/scss/selected_theme_variables';
+.cw-btn {
+  position: relative;
```

Comment:
what does `cw` refer ?

Reply:
nothing,it is just button I will change this